From 9145966b07d09605ad026c217825cabaee1a5dea Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Thu, 30 Jan 2014 17:06:42 +0100
Subject: [PATCH] builder: remove unused variables

Leftovers of the list_entries_short+list_entries_long split done in
commit 91aae893c70b3877b31803800ba77836fd7a45e8.

(cherry picked from commit bed8a168c0ed76e784adcc046256432f615e3314)
---
 builder/list_entries.ml | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/builder/list_entries.ml b/builder/list_entries.ml
index 7317cc7..27ea95e 100644
--- a/builder/list_entries.ml
+++ b/builder/list_entries.ml
@@ -48,9 +48,6 @@ let rec list_entries ~list_format ~sources index =
 and list_entries_short index =
   List.iter (
     fun (name, { Index_parser.printable_name = printable_name;
-                 size = size;
-                 compressed_size = compressed_size;
-                 notes = notes;
                  hidden = hidden }) ->
       if not hidden then (
         printf "%-24s" name;
-- 
1.8.4.2

